pages/welcome.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+
+import Container from '../components/Container';
+import { Box, Flex } from '../components/Grid';
+import Image from '../components/Image';
+import Link from '../components/Link';
+import Page from '../components/Page';
+import StyledCard from '../components/StyledCard';
+import StyledLink from '../components/StyledLink';
+import { useUser } from '../components/UserProvider';
+
+const messages = defineMessages({
+  welcomeMessage: {
+    defaultMessage: 'Welcome to Open Collective!',
+  },
+  welcomeDescription: {
+    defaultMessage:
+      'Now that you have created your personal account, there are a couple of things you can do from here...',
+  },
+  createCollectiveDescription: {
+    defaultMessage:
+      'Create a Collective to be able to accept donations, apply for grants, and manage your budget transparently.',
+  },
+  createOrgDescription: {
+    defaultMessage:
+      'Create a profile for your business to appear as financial contributor, enable your employees to contribute on behalf of your company, and more.',
+  },
+  engageHeading: {
+    defaultMessage: 'Contribute and engage with more Collectives',
+  },
+  engageDescription: {
+    defaultMessage:
+      'Discover active Collectives in the platform, contribute and engage with the communities that represent you.',
+  },
+  goToSettings: {
+    defaultMessage: 'Go to settings',
+  },
+  viewDocs: {
+    defaultMessage: 'View documentation',
+  },
+});
+
+const Welcome = () => {
+  const { formatMessage } = useIntl();
+  const { LoggedInUser } = useUser();
+  return (
+    <Page description={formatMessage(messages.welcomeMessage)}>
+      <Flex flexDirection={['column', 'row']} mb="61px" mt="112px" justifyContent="center" alignItems="center">
+        <Flex alignItems="center" mr={[0, '64px']} flexDirection={['column', 'row']}>
+          <Box>
+            <Image
+              src="/static/images/create-profile-page-logo.png"
+              alt="Open Collective logo"
+              height={160}
+              width={160}
+            />
+          </Box>
+          <Container pl="16px" pr={['16px', 0]} width={['100%', '404px']}>
+            <Flex fontSize="32px" fontWeight="700" color="black.900" lineHeight="40px">
+              {formatMessage(messages.welcomeMessage)}
```

Comment:
Why not just a `<FormattedMessage ... />`? I think we discussed this multiple times and decided that it should be the preferred approach for new developments (see https://github.com/opencollective/opencollective/blob/f9e85996c713672ff95656af0d499c90c0916462/rfcs/010-react-intl.md)

`defineMessages` is problematic because we can end up with unused strings and there's no way for us to know about them.

Reply:
Ops. I've corrected this to use only `<FormattedMessage>`.